src/documentation/pages/components/icon/_properties.vue

```diff
@@ -0,0 +1,81 @@
+<template lang="pug">
+  div
+    h2 Web Component
+
+    h3.mt--2 Properties
```

Comment:
@soujanreddy it is not neccesary anymore. Please review how it has been included in other components with properties.